### DISCUSSION TITLE
problem building IPA

### DISCUSSION THREAD

**Discussion by Rayene123** [codemagic-ci-cd/codemagic-docs#2511] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

i am building an ipa of my flutter app , i am using flutter version 3.19.4 , and am using the latest xcode version , knowing that i don't have an acutal mac device but i use the vm of codemagic , i included the code signing identities correctly of my app in the app store and the provisioning profile and the ios certificate . 

### Expected results

build successful and i find the ipa

### Actual results

build not successful and i get this error : Error (Xcode): Signing for "Runner" requires a development team. Select a development team in the Signing & Capabilities editor.
/Users/builder/clone/ios/Runner.xcodeproj

Encountered error while building for device.

### Build id (optional)

663a2e594fc7872406611e7c

**Comment by himesh-cm** [comment]
Hey @Rayene123, your project has incorrect bundle id, you should see below info if you expand the log section in your dashboard `Archiving com.example.pfeApp...`

While codesigning credentials you uploaded under `Teams > Settings > Code signing identities` are different.
Please make the required changes to your project and re-run the build on Codemagic.

**Reply by Rayene123** [reply]
i changed it and still the same problem

**Reply by himesh-cm** [reply]
Hey @Rayene123,

It seems like there may be an issue with your Flutter iOS workflow yaml. Please use our sample [yaml](https://github.com/codemagic-ci-cd/codemagic-sample-projects/blob/main/flutter/flutter-android-and-ios-yaml-demo-project/codemagic.yaml#L51) file for Flutter as a reference and ensure that all the steps are set up correctly. After making the necessary adjustments, kindly re-run the build and share the revised build ID. If you encounter any further issues, please let us know.

**Comment by himesh-cm** [comment]
I'm tentatively closing the discussion since we haven't heard back from you. You can open the discussion again by replying to this message or opening a new discussion.
